docs/en/contributor/Contributor-Getting-Started.md

```diff
@@ -1,69 +1,51 @@
 ---
 layout: global
-title: Contributing Tutorial
-nickname: Contributing Tutorial
+title: Contribution Guide
+nickname: Contribution Guide
 group: Contributor Resources
-priority: 0
+priority: 1
 ---
 
 * Table of Contents
 {:toc}
 
-Welcome to the Alluxio community! We are excited for your contributions and engagement in the
-community! Here is a guide on how to get started with becoming a member in the Alluxio open source
-community.
+# Introduction
```

Comment:
We have some css code to generate numbering for each heading, but it assumes that the headings start at level 2 (`##`).